GiniVision/Assets/de.lproj/Localizable.strings

```diff
@@ -120,3 +120,49 @@
 "ginivision.images.openWithTutorialStep1" = "openWithTutorialStep1";
 "ginivision.images.openWithTutorialStep2" = "openWithTutorialStep2";
 "ginivision.images.openWithTutorialStep3" = "openWithTutorialStep3";
+
+"ginivision.digitalinvoice.screentitle" = "Digital Invoice";
+"ginivision.digitalinvoice.paybuttontitle.noinvoice" = "Pay";
+"ginivision.digitalinvoice.paybuttontitle" = "Pay %d/%d";
+"ginivision.digitalinvoice.paybuttontitle.accessibilitylabel" = "Pay %d out of %d";
+
+"ginivision.digitalinvoice.items" = "Items %d/%d";
+"ginivision.digitalinvoice.items.accessibilitylabel" = "Items %d out of %d";
+"ginivision.digitalinvoice.whatisthisbutton" = "What is this?";
+
+"ginivision.digitalinvoice.whatisthisactionsheet.title" = "Is it helpful?";
```

Comment:
I see `ginivision.digitalinvoice.whatisthisactionsheet.title` in the title box when running the app. Nonetheless, 'Is it helpful?' doesn't seem right? Should it be 'What is this?' again, just like the button?

Reply:
Fixed.